src/en/space-station-14/combat.md

```diff
@@ -1,5 +1,70 @@
-```admonish warning "Attention: Placeholder!"
-This section is a placeholder, pending a design-doc being created by the related work-group
-```
+# Combat Design Document
 
-# Combat
+## Concept
+
+Combat in SS14 is a core part of the experience, acting as a way to exert influence in the game world on the same level of importance as speech and interactions. It is by far the most brute-force method of doing so as the main goal of combat is to forcibly deprive another party of resources, whether that be health, time, opportunity or equipment. Because of this it is a natural enabler (and sometimes solution) for conflict, creating an underlying system for consequences that reverberates throughout the game.
```

Comment:
Is it necessarily a "core part of the experience?" This seems to imply that you can't have fun without combat. Perhaps this should be amended to be "an important aspect of one of many play styles."

Reply:
> acting as a way to exert influence in the game world on the same level of importance as speech and interactions

This seems to glorify combat over other player-to-player interactions. We have rules for escalation, use it!